lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/util/qualitymetrics/MeanShiftQualityMetric.java

```diff
@@ -0,0 +1,41 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.util.qualitymetrics;
+
+/**
+ * Measures the relative shift of the mean of a value from the inlier to
+ * the outlier population.
+ */
+public class MeanShiftQualityMetric implements QualityMetric {
+    private int oCountIdx, iCountIdx, oMeanIdx, iMeanIdx;
+
+    public MeanShiftQualityMetric(
+            int oCountIdx,
+            int iCountIdx,
+            int oMeanIdx,
+            int iMeanIdx
+    ) {
+        this.oCountIdx = oCountIdx;
+        this.iCountIdx = iCountIdx;
+        this.oMeanIdx = oMeanIdx;
```

Comment:
I'm pretty sure this is also a sum / meanSum. Please make sure the naming is consistent otherwise it will get confusing what's actually a mean and what's actually a sum!